plugins/foreman_azure/2.1.x/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: plugin
+title: Foreman AzureRm 2.1.x Manual
+version: 2.1.x
+# uncomment for older versions than stable or nightly
+#robots: noindex
```

Comment:
Should this be commented for the older versions?

Reply:
done.